docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  llama-cpp:
+    build:
+      context: .
+    hostname: llama-cpp-server
+    command: "python3 -m llama_cpp.server --model /models/mistral-7b-instruct-v0.2.Q4_K_M.gguf --host 0.0.0.0 --port 8090"
+    stdin_open: true
+    tty: true
```

Comment:
```
volumes:
  - {MODELS_PTH:-./models}:/models
```

Reply:
interesting, that is some new syntax I'm not aware of

Reply:
Compose spec allows for simplified templating. This way it's possible to separate structure of the compose, from the variables. So you can override them when you need it. 

For example you can set TTY and stdin to be false by default, but override it CLI to do some debugging. 

There is bunch of other ways you can mess with composes, but I found this one to be the most useful.